src/applications/personalization/profile/tests/selectors.unit.spec.js

```diff
@@ -340,9 +340,13 @@ describe('profile selectors', () => {
   });
 
   describe('eduDirectDepositLoadError', () => {
-    it('returns the error if it exists', () => {
+    it('returns any non-403 errors that exist', () => {
```

Comment:
Thank you for adding these tests! 💯

Reply:
I don't know if I need additional comments somewhere to explain _why_ I added this special handling.

Reply:
I think your comment in the code `If the error is a 403 error, we will treat it like a no-data state, not an error` is sufficient!